pom.xml

```diff
@@ -150,7 +150,7 @@ flexible messaging model and an intuitive client API.</description>
     <gson.version>2.8.9</gson.version>
     <sketches.version>0.8.3</sketches.version>
     <hbc-core.version>2.2.0</hbc-core.version>
-    <cassandra-driver-core.version>3.6.0</cassandra-driver-core.version>
+    <cassandra-driver-core.version>3.7.2</cassandra-driver-core.version>
```

Comment:
it seems that cassandra-driver-core.version and cassandra.version can be merged into a single property, isn't it ?